### DISCUSSION TITLE
Ability to store an admin credential

### DISCUSSION THREAD

**Discussion by MiketheMaker18** [belowaverage-org/SuperLauncher#36] [Ideas]
Love SuperLauncher but it would be cool if you could configure your privileged account so it could automatically use those creds instead of manually entering.

**Comment by krisdb2009** [comment]
Super Launcher already does this! 😊

https://belowaverage.org/documentation/superlauncher/#:~:text=If%20%E2%80%9CUse%20credentials%20at%20startup%E2%80%9D%20is%20selected%2C%20next%20time%20Super%20Launcher%20starts%2C%20the%20account%20entered%20in%20the%20fields%20above%20will%20be%20used%20automatically.
